File: P2N_Pet_FrontEnd/P2N-Pet/projects/admin/src/app/models/age.ts
export class Age{
    public Id: number;
    public Title: string;
    public OrderView: number;
    public StatusText: string;
    public CreateUserName: string;
    public CreateDate: string;
    public UpdateUserName: string;
    public UpdateDate: string;
}

export class AgeCondition {
    Title: string;
    Status: string;
  
    constructor() {
      this.Title = '';
      this.Status = '0';
    }
  }

File: P2N_Pet_BackEnd/P2N_Pet_API/P2N_Pet_API/Database/PetShopModels/Age.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace P2N_Pet_API.Database.PetShopModels
{
    public partial class Age
    {
        public Age()
        {
            Petdetails = new HashSet<Petdetail>();
        }

        public ulong Id { get; set; }
        public string Title { get; set; }
        public int? Orderview { get; set; }
        public int? Status { get; set; }
        public ulong? Createuser { get; set; }
        public DateTime? Createdate { get; set; }
        public ulong? Updateuser { get; set; }
        public DateTime? Updatedate { get; set; }

        public virtual ICollection<Petdetail> Petdetails { get; set; }
    }
}
